Documentation/Diagnostics/PH2125.md

```diff
@@ -0,0 +1,65 @@
+# PH2125: Align number of + and == operators
+
+| Property | Value  |
+|--|--|
+| Package | [Philips.CodeAnalysis.MaintainabilityAnalyzers](https://www.nuget.org/packages/Philips.CodeAnalysis.MaintainabilityAnalyzers) |
+| Diagnostic ID | PH2125 |
+| Category  | [Maintainability](../Maintainability.md) |
+| Analyzer | [AlignOperatorCountAnalyzer](https://github.com/philips-software/roslyn-analyzers/blob/master/Philips.CodeAnalysis.MaintainabilityAnalyzers/Maintainability/AlignOperatorsCountAnalyzer.cs)
+| CodeFix  | No |
+| Severity | Error |
+
+## Introduction
+
+Users expect some operators to exists on the same type. For this diagnostic the plus (`+`) operator or the minus (`-`) operator is declared, a user will expect the equality operators (`==` and `!=`) to be available also. For some operators the pairing rule is mandated by [microsoft](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/operator-overloading#overloadable-operators). This diagnostics extends the pairing to the plus, minus and equalality operators.
+
+This diagnostic implements rule [7@531](https://csviewer.tiobe.com/#/ruleset/rule?setid=4T_Jr6-VSX6fp6egDIhGow&status=CHECKED,UNCHECKED&tagid=6u27MkXORKaev0VNuWR_SA&ruleid=TCjgzMnETo2yCNKdNzCS0Q) of the Philips C# [Coding Standard](https://csviewer.tiobe.com/#/ruleset/intro?tagid=6u27MkXORKaev0VNuWR_SA&setid=4T_Jr6-VSX6fp6egDIhGow).
```

Comment:
The other similar rules don't mention the standard?

Reply:
Correct, the others are not in the Philips Coding Standard